assets/stylesheets/common/common.scss

```diff
@@ -107,15 +111,33 @@
 }
 
 .post-info.activity-pub {
-  &.published .d-icon,
-  &.updated .d-icon {
-    color: var(--success);
-  }
-  &.scheduled .d-icon,
-  &.scheduled-past .d-icon {
+  display: flex;
+  align-items: center;
+  gap: 0.3em;
+
+  .d-icon {
     color: var(--primary-medium);
   }
-  &.deleted .d-icon {
+
+  &.published .d-icon-discourse-activity-pub,
+  &.updated .d-icon-discourse-activity-pub {
+    color: var(--success);
+  }
+  &.deleted .d-icon-discourse-activity-pub {
     color: var(--danger);
   }
+
+  .activity-pub-visibility .d-icon {
+    font-size: 0.85em;
+  }
+}
+
+.activity-pub-visibility-dropdown {
+  .select-kit-header {
+    padding: 4px 6px;
```

Comment:
We usually try to avoid modifiying the styling of an individual dropdown, because it then no longer aligns / matches the general core styles. Especially here, I don't think this warrants any style changes.

Reply:
We'll change the component as you suggest and remove these.

Reply:
Done https://github.com/discourse/discourse-activity-pub/commit/d6565a4cac9d8f17a51c083601e4256728dd08df